style.css

```diff
@@ -658,105 +715,443 @@ body {
-    font-style: normal;
     font-weight: 700;
     font-size: 20px;
     line-height: 20px;
     justify-content: center;
     align-items: center;
+    margin-left: 2vh;
   }
 
   .body-2 {
-    padding-top: 10%;
-    padding-bottom: 10%;
-    width: 50%;
-    margin-left: 2%;
     display: flex;
     flex-direction: row-reverse;
-    justify-content: space-around;
-    align-items: flex-start;
+    align-items: center;
+    padding-top: 10vh;
+    margin-left: 2vh;
+    padding-left: 0%;
+    padding-bottom: 18vh;
+    width: 50%;
     background-image: none;
   }
 
-  .txt {
+  #sm {
     display: flex;
-    width: 75%;
     flex-direction: column;
+    justify-content: space-between;
+    padding-top: 5%;
+    width: 200px;
+    height: 40px;
+    margin-bottom: 30%;
     align-items: flex-start;
-    justify-content: space-around;
+    padding-right: 0%;
+    margin-right: 0%;
   }
 
-  .h1 {
-    font-family: "Crete Round", serif;
+  .txt {
+    padding-left: 5vh;
+    margin-left: 0%;
+  }
+
+  .h1 p {
+    padding-bottom: 0%;
+    margin-bottom: 0%;
+    font-family: "Crete Round", sans-serif;
     font-style: normal;
     font-weight: 400;
     font-size: 72px;
     line-height: 82px;
-
-    /* or 113% */
-
     color: #f55800;
   }
 
   .h2 {
-    padding-top: 3%;
+    padding-top: 0%;
+    margin-top: 0%;
+    padding-bottom: 0%;
+    margin-bottom: 0%;
     font-family: "Inter", sans-serif;
     font-style: normal;
     font-weight: 500;
-    font-size: 28px;
+    font-size: 32px;
     line-height: 44px;
-
-    /* or 138% */
-
     display: flex;
     align-items: center;
     color: #fff;
   }
 
+  .h2 p {
+    padding-top: 0%;
+    margin-top: 0%;
+    padding-bottom: 0%;
+    margin-bottom: 0%;
+  }
+
   .h3 {
-    padding-top: 3%;
+    width: 80%;
+  }
+
+  .h3 p {
     font-family: "Inter", sans-serif;
     font-style: normal;
     font-weight: 400;
     font-size: 20px;
     line-height: 32px;
+    color: #f4f5f7;
+  }
+
+  ul {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
 
-    /* or 160% */
+  #title2 {
+    display: flex;
+    width: 80%;
+    align-items: center;
+    margin-top: 5vh;
+  }
 
-    /* Neutrals/N20 */
+  #title2 h1 {
+    width: 38%;
+    padding-right: 0%;
+    margin-right: 0%;
+    align-self: center;
+    margin-top: 20px;
+    padding-bottom: 0%;
+  }
 
-    color: #f4f5f7;
+  #title2 hr {
+    flex-grow: 1;
+    padding-left: 0%;
+    margin-left: 0%;
+    margin-right: 0%;
+    align-self: center;
+    padding-top: 0%;
+    padding-bottom: 0%;
+    border-top: 1px solid black;
   }
 
-  #sm {
-    width: 20%;
-    height: 100%;
+  #works {
+    display: flex;
+    width: 85%;
+    align-items: flex-start;
+    flex-direction: row;
+    margin-bottom: 0%;
+    padding-bottom: 0%;
+    margin-top: 0%;
+  }
+
+  .container {
+    margin: 0 2vh;
+    padding: 0% 0%;
+    display: flex;
+    width: 150%;
+    flex-direction: column;
+    align-items: flex-start;
+  }
```

Comment:
As mentioned above, kindly I suggest you stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

> ## The Figma design
>
><img width="1440" alt="Screenshot 2023-06-15 at 11 36 09 PM" src="https://github.com/Rdzaman20000/Portfolio/assets/91237525/9e47fa0b-82ac-424b-aa8e-de969c3f3f76">
><img width="1440" alt="Screenshot 2023-06-15 at 11 36 33 PM" src="https://github.com/Rdzaman20000/Portfolio/assets/91237525/900323d6-6401-485f-b76c-c9b958b15b71">
>
> ## Your Design
>
><img width="1440" alt="Screenshot 2023-06-15 at 11 08 33 PM" src="https://github.com/Rdzaman20000/Portfolio/assets/91237525/e56dc25f-2411-44b4-b2f1-cdc602afbb98">
><img width="1440" alt="Screenshot 2023-06-15 at 11 10 25 PM" src="https://github.com/Rdzaman20000/Portfolio/assets/91237525/b77d54cf-3faf-43c4-9e16-2393eb086fca">